docs/docs/run-source.rst

```diff
@@ -3,24 +3,27 @@ Run Klaw from source
 
 Klaw is a Java project, you will need `Maven <https://maven.apache.org/>`_ installed to run this project from source.
 
-1. Download the latest code from both repositories:
+1. Download the latest code from the below repository:
 
    * https://github.com/aiven/klaw
-   * https://github.com/aiven/klaw-cluster-api
 
-   .. tip:: You can also clone both repositories to use the "bleeding edge" version if you wish.
+   .. tip:: You can also clone repository to use the "bleeding edge" version if you wish.
 
-2. Build both projects by running ``mvn clean package`` from the top level of each project directory. This will build a JAR file in the ``target/`` directory of each project.
+2. Configure Cluster Api access
+    - Configure the property ``klaw.clusterapi.access.base64.secret`` with a base64 string in module 'core' (application.properties)
+    - Configure the property ``klaw.clusterapi.access.base64.secret`` with the above base64 string in cluster-api (application.properties) module as well.
+
+2. Build the project by running ``mvn clean package`` from the top level of the project directory. This will build JAR files in the ``target/`` directories of each modules 'core' and 'cluster-api'.
```

Comment:
this implies that a person already has installed `mvn`
in case there is no installed `mvn` it's better to suggest `./mvnw clean package`
or `mvnw clean package` for windows users

Reply:
Agree, updated.